doc/logging.md

```diff
@@ -0,0 +1,24 @@
+# How does Happy test log work
+
+## Types of Happy logs
+  * Happy host logs — Happy will run as host, Happy host logs will have logs from Happy tool only.
+
+  * Happy device logs — Happy will host virtual network which has multiple virtual devices, Happy device logs will have logs for software running on the virtual device, openweave is one of the software example, so device logs will have openweave logs.
```

Comment:
When we mention "openweave" we should be consistent in our syntax, for example "openweave" vs. "openweave-core".

Reply:
This is tricky because if I were referring to the repo itself I'd just use openweave-core... but if we're talking about the high-level "product" is there a difference between OpenWeave and OpenWeave Core?  If there is no meaningful difference I think we should just use OpenWeave for the product in general and openweave-core when referring to the repo specifically in code examples.

Reply:
How about we use OpenWeave in all places